### DISCUSSION TITLE
всем привет

### DISCUSSION THREAD

**Discussion by Charaty4345** [vvip-68/GyverPanelWiFi#260] [Задавайте вопросы]
всем привет
при первом подключении  телефона к герлянде появляются настройки (выбор угла подключения и кол-во диодов)
как сделать чтоб эти настройки появились опять? (я меняю кол-во диодов в матрице в скетче .но  при подключении с телефона . остаются старые настройки)

**Comment by vvip-68** [comment]
А вы точно про этот проект пишете? Там нет никакого автоматического появления настроек при первом подключении.
Там есть специальное приложение PanelSetupWiFi, которое что при первом подключении, что при последующих позволяет изменить настройки (размер, тип, угол подключение, направление из угла) матрицы. Если же вы используете какой-то другой проект - тут мы ничего подсказать не сможем.

Настройки измененные в скетче применяются только когда заливка прошивки идет с включенной опцией компиляции очищать весь EEPROM (Erase All Flash Content). Если это не включено - настройки сохраненные в энергонезависимую память не стираются и при последующих запусках используются именно они, а не те что изменены в скетче.

**Reply by Charaty4345** [reply]
Спасибо вчера нашел . у меня не была включена опция Erase All Flash Content
